bundles/org.openhab.binding.tr064/README.md

```diff
@@ -35,48 +37,97 @@ For security reasons it is highly recommended to set both, username and password
 
 ### `fritzbox`
 
-All additional parameters for `fritzbox` devices (i.e. except those that are shared with `generic`) are advanced parameters.
+The `fritzbox` devices can give additional informations in dedicated channels, controlled  by additional parameters, w.r.t. to `generic` devices.
+These parameters are advanced parameters, i.e. they appear in the Main User Interface only if the `Show Advanced` option is selected.
```

Comment:
I think you can remove this line.

Reply:
ok, I removed the sentence, but left a mention of the show advanced in the main ui.